codestodians/food-basket-api
# [Bug: #154398686] Fix schema logic

#### What does this PR do?
- Optimizes the logic in selected schema based on observation

#### Description of Task to be completed?
- Update `user`, `order` and `product` schema.
- Add extra test cases

#### How should this be manually tested?
- Check out into the `bg-fix-schema-logic-154398686` branch and check that the extra logic added in the files in this PR work as expected.

#### What are the relevant pivotal tracker stories?
- [#154398686](https://www.pivotaltracker.com/story/show/154398686)
